inject-generator/src/main/java/io/avaje/inject/generator/MethodReader.java

```diff
@@ -229,9 +244,11 @@ void builderBuildAddBean(Append writer) {
       }
       String indent = optionalType ? "        " : "      ";
       writer.indent(indent);
-      if (hasLifecycleMethods()) {
-        writer.append("var $bean = ");
```

Comment:
We have to keep using the returned `$bean` because with testing and using mockito the `$bean` instance can be different to `bean`. 

So this change breaks that.

Reply:
I see, I'll probably add a test for that later.